Benchmark-Models/Laske_PLOSComputBiol2019/observables_Laske_PLOSComputBiol2019.tsv

```diff
@@ -0,0 +1,14 @@
+observableId	observableFormula	noiseFormula	observableTransformation	noiseDistribution
+RM5	R_M5	noiseParameter1_RM5	log	normal
+RVSegTot	vRNA_offset + R_V_seg_tot	noiseParameter1_RVSegTot	log	normal
+RCSegTot	R_C_seg_tot	noiseParameter1_RCSegTot	log	normal
+Vrel	V_rel	noiseParameter1_Vrel	log	normal
+IntNucOffset	Int_nuc_off	13.0334748500416 * scale_sigma_FracNucInt	lin	normal
+FracNucInt_1	F_rnp_nuc + Int_nuc_off	3.10956052629092 * scale_sigma_FracNucInt	lin	normal
```

Comment:
what are the numerical values in the noise formulas?? could you please explain

Reply:
The numerical values are the standard deviations of the measurements of FracNucInt.

Reply:
thanks for explaining, I have another question 😄 
why do you need then the scaling parameter for the noise instead of using directly the measured noise as fixed? was this also done in the original publication?

Reply:
This just gives the model more flexibility for optimization. Was not done this way in the original publication. Laske et al. did not estimate any noise parameters (see section "Parameter estimation" in the publication).

Reply:
Just Gaussian noise model did not work, so we tried using the experimental standard deviations reported by the authors. I was (am) not super happy of this (don't trust the values too much with these low number of replicates), so I suggested to add a scaling parameter to add a little bit of freedom. If you think this is too much non-standard we can take it away and try refitting. It will probably not change much. (@C-Peiter what was the CI for the scaling parameter?)

Reply:
CI was 0.5-1.1

Reply:
may i ask what did not work? just fixing the noise values to the measured ones? or estimating a single `sigma` for this specific time course?

Reply:
I think, we are getting two things mixed up here. Gaussian noise did not work when we tried to optimize with another observable formula for FracNucInt, which is why we changed the noise model. I will have a look into this and also try refitting with normal noise for the authors' observable formula and get back to you.

Reply:
I reoptimized with a single sigma and it works fine. Would you prefer this over the original noise model? Then I could change it.